structure/0007-community-team.md

```diff
@@ -227,9 +214,15 @@ the administrative board must be invited to vote on the poll, with their votes c
 voted, they may change their vote or retract it, and their final vote will also count for three. The members of the
 administrative board are not required to vote if they don't feel that it's necessary.
 
-If any active voting members of the Community Team don't vote on the poll (unintentionally or otherwise), their vote **must be
-counted as an abstention.** If a member is inactive (has not interacted with community management spaces for more than two weeks),
-they will not be counted as a positive vote, negative vote or abstention, and will not be counted in the tally for voting.
+If any active voting members of the Community Team don't vote on the poll (unintentionally or otherwise), their vote
+**must be counted as an abstention.** If a member is inactive (as defined below), they will not be counted as a
+positive vote, negative vote, or abstention, and will not be counted in the tally for voting.
+
+When taking into consideration plural systems, one system is treated as one voting bloc, regardless of individual
+members, Discord accounts, or other factors. This is to ensure that a large system is not able to overpower the votes
+of other Community Team members. If multiple votes are tallied on one poll originating from one system, all of the
+system's votes will be removed and they will be asked to vote again. In the case that all of these multiple votes are
+for the same option, all but one of the reactions may be removed instead.
```

Comment:
Use "votes" instead of "reactions" to keep this applicable to any systemic changes we need to make later.